app/pages/home.cjsx

```diff
@@ -42,17 +42,27 @@ counterpart.registerTranslations 'en',
 FeaturedProjects = React.createClass
   displayName: "FeaturedProjects"
 
+  imagePromise: (project) ->
+    src = if project.avatar_src
+      "//#{ project.avatar_src }"
+    else
+      '/assets/simple-avatar.jpg'
+    Promise.resolve src
+
   render: ->
     <div className="featured-projects">
-      <PromiseRenderer promise={apiClient.type('projects').get(FEATURED_PRODUCT_IDS)}>{(projects) =>
+      <PromiseRenderer promise={apiClient.type('projects').get(id: FEATURED_PRODUCT_IDS, cards: true)}>{(projects) =>
```

Comment:
will this use the index route? If it's a show route, i'll have to make the cards context expandable to index / show.